build.sh

```diff
@@ -89,8 +94,8 @@ build()
       #OSX
       if [ "$ARCH" == "x86_64" ]; then
          ${SRC_DIR}/Configure darwin64-x86_64-cc --prefix="${BUILD_DIR}/${OPENSSL_VERSION}-${ARCH}" &> "${BUILD_DIR}/${OPENSSL_VERSION}-${ARCH}.log"
-         sed -ie "s!^CFLAG=!CFLAG=-isysroot ${SDK} -arch $ARCH -mmacosx-version-min=${OSX_DEPLOYMENT_VERSION} !" "${SRC_DIR}/Makefile"
-         sed -ie "s!^CFLAGS=!CFLAGS=-isysroot ${SDK} -arch $ARCH -mmacosx-version-min=${OSX_DEPLOYMENT_VERSION} !" "${SRC_DIR}/Makefile"
+         sed -ie "s!^CFLAG=!CFLAG=-isysroot ${SDK} -arch $ARCH -mmacosx-version-min=${OSX_DEPLOYMENT_VERSION} -fembed-bitcode !" "${SRC_DIR}/Makefile"
```

Comment:
o, can you confirm that bitcode works now?

Reply:
> can you confirm that bitcode works now?

Actually... no, it does not seem to work for me 😢 I don’t have an actual physical device, but the library fails to compile with `carthage build` due to some bitcode issues.

Reply:
do you think we should revert this? can you PR?

Reply:
Weird, because it _worked_ for me when I ran `./build.sh` and pushed the resulting binaries into a tag in my own fork. I’ll try that again with the current master.

Maybe it does not work because you and I have different Xcode versions (because compilation errors indicate bitcode version mismatch). But in that case I fail to understand the reason for bitcode embedding in the first place, if it’s _that tightly_ coupled with the toolchain being used. Does Apple provide system frameworks compiled against all Xcode versions? Should developers distribute their frameworks compiled against all Xcode versions?..